package.json

```diff
@@ -41,9 +47,21 @@
       "types": "./_types/index.d.ts"
     }
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,json,md,ts,}": "prettier --write"
+  },
   "module": "./_esm/index.js",
   "prettier": {
-    "semi": false
+    "arrowParens": "avoid",
+    "quoteProps": "consistent",
+    "semi": false,
+    "singleQuote": true,
```

Comment:
Thinking about `singleQuote` and `semi`... Resistance is futile. We may want to surrender to Prettier's defaults.

But not for `quoteProps` as `consistent` seems the way to go. I still think Prettier gets this wrong but history may prove I am.